scapy/contrib/automotive/doip.py

```diff
@@ -68,7 +72,7 @@ class DoIP(Packet):
             0x0d: "Reserved by ISO 13400", 0x0e: "Reserved by ISO 13400",
             0x0f: "Reserved by ISO 13400",
             0x10: "Routing activation required to initiate central security",
-        }), lambda p: p.payload_type in [4]),
+        }), lambda p: cast(DoIP, p).payload_type in [4]),
```

Comment:
Please don't put these casts. This isn't scalable. We will tweak mypy rules so that we never have to put this many in layers/contribs.

Reply:
fixed it